LDAR_Sim/model_code/default_parameters/default_program_parameters.py

```diff
@@ -0,0 +1,61 @@
+# ------------------------------------------------------------------------------
+# Program:     The LDAR Simulator (LDAR-Sim)
+# File:        default_program_parameters
+# Purpose:     Default program parameters
+#
+# Copyright (C) 2018-2020  Thomas Fox, Mozhou Gao, Thomas Barchyn, Chris Hugenholtz
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the MIT License as published
+# by the Free Software Foundation, version 3.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# MIT License for more details.
+
+# You should have received a copy of the MIT License
+# along with this program.  If not, see <https://opensource.org/licenses/MIT>.
+#
+# ------------------------------------------------------------------------------
+
+default_program_parameters = {
```

Comment:
This is awesome!!
I know we talked about this - but why not just combine this with the default global parameters if they are defaults? Wouldn't that have the same outcome/effect and result in fewer files and arbitrary distinction between what a "global" and "program" parameter is? For example, if I create a new parameter, how do I know if I create a default in 'global' or 'program'?

Reply:
@KeeganShaw-GIS  can maybe provide guidance here?
I think anything in ldar_sim_main.py that is outside of the starmap would be considered constant among all programs and thus is better to be explicitly called a global parameter.

If has potential to be different among programs, it makes sense to be a program parameter.

I'm not too picky, but it does feel that having to make sure the number of timesteps is the same among all the programs is a lot of work and error prone.

There are probably some parameters that are better globals right now - but I've left these changes for discussion as they'd require coding in the model.

Reply:
I agree with leaving them as Globals. They can still be overwritten at the program level though right?

Reply:
Leaving for now.

Reply:
Changed to issue to maybe fix in the future #92